jq/jv.go

```diff
@@ -230,7 +230,7 @@ func JvFromJSONString(str string) (*Jv, error) {
 // JvFromJSONBytes takes a utf-8 byte sequence containing JSON and returns the
 // jv representation of it.
 func JvFromJSONBytes(b []byte) (*Jv, error) {
-	jv := C.jv_parse((*C.char)(unsafe.Pointer(&b[0])))
+	jv := C.jv_parse_sized((*C.char)(unsafe.Pointer(&b[0]), C.int(len(b))))
```

Comment:
```suggestion
	jv := C.jv_parse_sized((*C.char)(unsafe.Pointer(&b[0])), C.int(len(b)))
```
I had to move the closing parenthesis in order for this to compile on my machine.

Reply:
Thanks!  I hadn't gotten around to looking at it, and didn't even have a go dev env set up so was using a few spare moments and CI to try and fix it!

Reply:
No worries. I just submitted #16 which fixes the broken test that I'm seeing in this CI run.

Your CI doesn't seem to run on PRs from outside contributors, which makes it harder to demonstrate that this fixes the build 😄  and is also how the failing test got merged to begin with

Reply:
Curious, I wonder how I did that. Given it's just CircleCi that wasn't my intent :D

Reply:
Fixed. I had somehow turned that setting off in CircleCI.